bench_autoawq/README.md

```diff
@@ -6,31 +6,56 @@
+| During the Manhattan Project, John von Neumann suggested how to maximize the destruction caused by a detonating bomb. Which was his idea? Answer as much precise as possible with one sentence.                                                                                                                                                                                                                                 | John von Neumann proposed the concept of a "fusion bomb," which involved using nuclear fission to create a chain reaction that would release a massive amount of energy from a single, critical mass of fissile material, leading to an unprecedented level of destruction.                                                                                 | John von Neumann proposed the concept of a "fusion bomb," which would achieve maximum destruction through the simultaneous detonation of a fission bomb and a fusion reaction, creating a chain reaction of nuclear reactions that would release an enormous amount of energy. This idea was later developed into the hydrogen bomb.                                                                                                                                                                                                             |
+
+## 👀 Some points to note:
 
 1. AutoAWQ is not supported devices other than GPU (only supports when CUDA is available).
 2. We are independently benchmarking AutoAWQ (i.e. the actual AWQ quantization method here). We are not benchmarking with combinations like: AutoAWQ + VLLM or AutoAWQ + TensorRT.
 3. For doing this benchmark, the default model that was choosen was: [Llama2-AutoAWQ by The Bloke](https://huggingface.co/TheBloke/Llama-2-7B-AWQ)
-4. AutoAWQ does not support INT8 quantization properly yet. See [this issue](https://github.com/casper-hansen/AutoAWQ/issues/45).
+4. AutoAWQ does not support INT8 quantization properly yet. See [this issue](https://github.com/casper-hansen/AutoAWQ/issues)
```

Comment:
the issue has been closed?

Reply:
Actually this still remains incomplete from this PR: https://github.com/casper-hansen/AutoAWQ/pull/71